api/app.go

```diff
@@ -73,14 +74,15 @@ func (a *App) getRouter() *mux.Router {
 		&VersionMiddleware{},
 	)).Methods("GET").Name("game")
 
-	r.Handle("/games", Chain(
+	r.HandleFunc("/games/{id}", Chain(
 		&GameHandler{App: a, Method: "upsert"},
 		&MetricsReporterMiddleware{App: a},
 		&NewRelicMiddleware{App: a},
 		&LoggingMiddleware{App: a},
 		&VersionMiddleware{},
+		NewParamKeyMiddleware(a, func(id string) bool { return govalidator.Matches(id, "^[^-][a-z0-9-]*$") }),
 		NewValidationMiddleware(func() interface{} { return &models.Game{} }),
-	)).Methods("PUT").Name("game")
+	).ServeHTTP).Methods("PUT").Name("game")
 
 	r.Handle("/offer-templates", Chain(
```

Comment:
We forgot to chance this route after the rebase :)
Just replace /offer-templates by /templates and it will be ok.